tests/Rules/Doctrine/ORM/data/MyEntity.php

```diff
@@ -19,6 +19,12 @@ class MyEntity
 	 */
 	private $id;
 
+	/**
+	 * @ORM\Column(type="integer")
+     * @ORM\Version
```

Comment:
We currently don't have a test-suite for entities with attributes. How would you test this?

Reply:
I've found out that removing the `propertyAlwaysWrittenTags` does not make the tests. So blindly adding this property here is also not a good way to test it. I'm not sure how to approach this because there is no `Rule` I could refer to in test based on the `RuleTestCase`.